%3a14/char-sets.sls

```diff
@@ -19,7 +17,7 @@
     list->char-set! string->char-set!
     char-set-filter  ucs-range->char-set
     char-set-filter! ucs-range->char-set!
-    ->char-set
+    (rename (x->char-set ->char-set))
```

Comment:
Why the shift in names?

Reply:
The name used by SRFI-14 is `->char-set`, so we are exporting the correct name. Scheme 48, however, calls it `x->char-set` because `->char-set` is not a valid R5RS identifier.